frontend-react-app/src/components/PaginationComponent/Pagination.js

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import './Pagination.scss';
+
+const Pagination = ({ setOffset, limit, countAllReception, setUseEffectDo }) => {
+  const countPages = Math.ceil(countAllReception / limit);
+
+  const arrNunPage = [];
+  for (let i = 0; i < countPages; i++) {
+    arrNunPage.push(i);
+  }
+
+  const changePage = (element) => {
+    setOffset(element);
+    setUseEffectDo(true);
+  }
+
+  return (
+    <nav>
+      <ul className="pagination">
+        {
+          arrNunPage.map((element, index) => (
```

Comment:
you view users button "last", "first", "next", "prev"?

Reply:
no, only numbers pages